tools/configure.d/nodedownload.py

```diff
@@ -38,6 +38,9 @@ def retrievefile(url, targetfile):
         ConfigOpener().retrieve(url, targetfile, reporthook=reporthook)
         print ''  # clear the line
         return targetfile
+    except IOError as err:
+        print ' ** IOError %s\n' % err
```

Comment:
Please put __from \_\_future\_\_ import print_function__ at the top of this file above other imports and then use the __print()__ function instead of the __print__ statement to be ready for Python 3.  There are 437 days until Python 2 end of life.